rio_stac/scripts/cli.py

```diff
@@ -66,14 +66,27 @@ def _cb_key_val(ctx, param, value):
 @click.option(
     "--with-proj/--without-proj",
     default=True,
-    help="Add the projection extension and properties (default to True).",
+    help="Add the 'projection' extension and properties (default to True).",
 )
 @click.option(
     "--with-raster/--without-raster",
     default=True,
-    help="Add the raster extension and properties (default to True).",
+    help="Add the 'raster' extension and properties (default to True).",
+)
+@click.option(
+    "--with-eo/--without-eo",
+    default=True,
+    help="Add the 'eo' extension and properties (default to True).",
```

Comment:
Can use `show_default=True` instead of fixed text:
![image](https://user-images.githubusercontent.com/58314/188207080-c1c764a8-8711-4083-a0be-aa830d0a991c.png)